app/assets/javascripts/student_profile/__snapshots__/IncidentHeatmap.test.js.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`snapshots 1`] = `
+<div
+  className="DashboardScatterPlot"
+  id="string"
+  style={
+    Object {
+      "display": "flex",
+      "flex": 1,
+      "padding": 10,
+      "width": "100%",
+    }
+  }
+>
+  <div
```

Comment:
This snapshot isn't testing much since `HighchartsWrapper` doesn't render much itself directly.  If you add in `jest.mock('../components/HighchartsWrapper', () => 'highcharts-wrapper');` before the `it` calls in the test, that will mock the component, and then snapshot test will verify that `<HighchartsWrapper />` is called with the same arguments over time.

If you add that in and then run `yarn jest app/assets/javascripts/student_profile/IncidentHeatmap.test.js` you can see the diff and how the snapshot becomes more useful in checking that the way the component is called stays the same over time.

Reply:
Ah, thanks for catching that!